contracts/upgradeable_contracts/BridgeValidators.sol

```diff
@@ -9,89 +9,167 @@ import "../upgradeability/EternalStorage.sol";
 contract BridgeValidators is IBridgeValidators, EternalStorage, Ownable {
     using SafeMath for uint256;
 
+    address public constant F_ADDR = 0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF;
+
     event ValidatorAdded (address indexed validator);
```

Comment:
should we extend the event to report reward address as well?

Reply:
Done https://github.com/poanetwork/poa-bridge-contracts/pull/130/commits/5c553e06eb323e77c3628549c38f04bf224e07a0